base/applications/osk/keyboard.c

```diff
@@ -0,0 +1,251 @@
+/*
+ * PROJECT:         ReactOS On-Screen Keyboard
+ * LICENSE:         GPL - See COPYING in the top level directory
+ * PURPOSE:         Keyboard layout data.
+ * COPYRIGHT:       Baruch Rutman (peterooch at gmail dot com)
+ * 
+ * This is partialy based on the original en-US dialog templates.
+ */
+#include "precomp.h"
+
+KEY EnhancedKeyboardKeys[] = 
```

Comment:
I am not sure if you're aware or not but certain keys like ALT are localized, depending on the locale selected within the system and whatnot. Right now you're hardcoding these keys and regardless of what locale you choose for the main keyboard keys, the rest of the keys are still remained as is. Is that intended?

Reply:
I have noticed that some keys are translated in some translations, I guess I could add more string entries, although if there is a way to extract it from the keyboard layout would be better